database/postgresSetup.sql

```diff
@@ -32,7 +32,7 @@ CREATE TABLE IF NOT EXISTS events(
       user_id int REFERENCES users,
       PRIMARY KEY (feed_id, user_id));
 
-INSERT INTO users(username, password) values ('foo', 'bar');
+INSERT INTO users(username, password) values ('foo', crypt('bar', gen_salt('md5')));
```

Comment:
Right, I know what encrypting and salting are for but the file is just called `database/postgresSetup.sql` and it isn't clear from context whether populating the database with mock data is done during testing or app startup. If this is the real database it shouldn't start with anything in it. (Attackers looking at this project -- it's open source after all -- can see there is a foo/bar user and easily break into a Grapevine installation.) If the mock data is created only during testing then that's okay.